src/Core/Settings.h

```diff
@@ -233,7 +233,7 @@ class IColumn;
     M(Bool, do_not_merge_across_partitions_select_final, false, "Merge parts only in one partition in select final", 0) \
     M(Bool, split_parts_ranges_into_intersecting_and_non_intersecting_final, true, "Split parts ranges into intersecting and non intersecting during FINAL optimization", 0) \
     M(Bool, split_intersecting_parts_ranges_into_layers_final, true, "Split intersecting parts ranges into layers during FINAL optimization", 0) \
-    M(Bool, allow_experimental_inverted_index, false, "If it is set to true, allow to use experimental inverted index.", 0) \
+    M(Bool, allow_experimental_inverted_index, false, "If it is set to true, allow to use experimental fulltext (inverted) index.", 0) \
```

Comment:
As the feature is experimental and we are changing the declaration of them I think we can also change the settings. If somebody didn't use the feature, they won't have any issues. If somebody uses the feature, then they will have bigger problems with metadata and stuff than a single setting name.

Reply:
You are generally right. But: I believe (not 100% sure) that setting `allow_experimental_inverted_index` is force-disabled in ClickHouse cloud via config file. Renaming the setting name will cause unnecessary trouble :-/ At some point, the setting will go away so `allow_experimental_inverted_index` should be good for the time being.